Formula/osmosis.rb

```diff
@@ -13,6 +13,32 @@ def install
     bin.install "bin/osmosis"
     libexec.install %w[lib config script]
   end
+
+  test do
+    path = testpath/"test.osm"
+    path.write <<-EOS
```

Comment:
Use EOS.undent` and intent the below lines to be two-spaces beyond the whitespace on this line.